src/pybind/mgr/dashboard/frontend/src/app/app-routing.module.ts

```diff
@@ -335,10 +333,32 @@ const routes: Routes = [
     component: BlankLayoutComponent,
     children: [
       // Single Sign-On (SSO)
-      { path: 'sso/404', component: SsoNotFoundComponent },
+      {
+        path: 'sso/404',
+        component: ErrorComponent,
+        data: {
+          errorHeader: $localize`User Denied`,
+          errorMessage: $localize`Sorry, the user does not exist in Ceph.
+                You'll be logged out from the Identity Provider when you retry logging in.`
+        }
+      },
       // System
-      { path: '403', component: ForbiddenComponent },
-      { path: '404', component: NotFoundComponent },
+      {
+        path: '403',
+        component: ErrorComponent,
+        data: {
+          errorHeader: $localize`Access Denied`,
+          errorMessage: $localize`Sorry, you don’t have permission to view this page or resource.`
+        }
+      },
+      {
+        path: '404',
+        component: ErrorComponent,
+        data: {
+          errorHeader: $localize`Page not found`,
+          errorMessage: $localize`Sorry, we couldn’t find what you were looking for.`
+        }
+      },
       { path: '**', redirectTo: '/404' }
     ]
```

Comment:
I think we should just simply send all errors to the unique error URL:
```suggestion
      },
      { path: 'error', component: ErrorComponent },
      { path: '**', redirectTo: '/error' , data: DashboardNotFoundError },
    ]
```